receiver/vcenterreceiver/metadata.yaml

```diff
@@ -382,3 +382,19 @@ metrics:
       aggregation: cumulative
     attributes: []
     extended_documentation: As measured over the most recent 20s interval.
+  vcenter.vm.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the VM.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.vm.cpu.usage:
+    enabled: true
+    description: The amount of CPU in Hz used by the VM.
+    unit: "MHz"
```

Comment:
Are you certain this unit makes sense? This is a measure of frequency (`cycles/second`). Often CPU usage is reported in `cycles`.

Reply:
https://vdc-repo.vmware.com/vmwb-repository/dcr-public/1ef6c336-7bef-477d-b9bb-caa1767d7e30/82521f49-9d9a-42b7-b19b-9e6cd9b30db1/vim.vm.Summary.QuickStats.html

The `overallCpuUsage` field of this object in the VMware ObjectManager is what is being reported, so I think its accurate for MHz being the unit just based off this but just figured I'd chime in!

Reply:
I actually took the blocks from the host cpu usage and utliization in metadata.yaml, just changing host to vm. I checked various docs and megahertz is used across all for VM cpu usage. The vshpere client graph also shows _CPU usage in megahertz during the  interval: 1,234_.